src/psyclone/tests/test_files/dynamo0p3/1.6.2_single_invoke_1_int_from_derived_type.f90

```diff
@@ -14,7 +14,7 @@ program single_invoke
   ! In the third the type-bound routine takes an argument and in the
   ! fourth this argument is itself obtained by dereferencing another
   ! derived type.
-  use testkern_one_int_scalar, only: testkern_type
+  use testkern_one_int_scalar_mod, only: testkern_one_int_scalar_type
```

Comment:
Please update the copyright notice in this file.

Reply:
Done.